index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Music concert Home Page</title>
+    <link rel="stylesheet" type="text/css" href="index.css">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <script src="https://kit.fontawesome.com/0f56b98a6f.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <div class="header1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#" class="logout">Logout</a>
+        </div>
+        <div class="header2">
+        <div class="header-logo">
+            <img src="Images/white logo.png">
+        </div>
+        <div>
+            <button class="hambuger">
+                <i class="fa-solid fa-bars"></i>
+            </button>
+        </div>
+        <nav class="nav-menu">
+            <button class="close">
+                <i class="fa-solid fa-xmark"></i>
+            </button>
+            <ul>
+                    <li><a href="About.html" class="item-about">About</a></li>
+                    <li><a href="#" class="item-program">Program</a></li>
+                    <li><a href="#" class="item-join">Join</a></li>
+                    <li><a href="#" class="item-sponsor">Sponsor</a></li>
+                    <li><a href="#" class="item-newa">News</a></li>
+                   
+                </ul>
+                </nav>
+            <button class="campaign">
+                Referral program
+            </button>
+        </div>
+    </header>
+    <section class="home-about">
+        <div class="home2">
+        <h3>"Join the Musical Celebration, Experience Unity!"</h3>
+        <h1>HARMONY IN MOTION 2023</h1>
+        <p class="description">Get ready for the musical event of the year, "Harmony in Motion 2023"! This musical extravaganza will bring together the beats of various genres to create a symphony of sound. Join us on the evening of August 20, 2023 , for an unforgettable night of unity and rhythm. As we come together to celebrate the power of music, let's create memories that will last a lifetime</p>
+        <p class="date">
+            2023.8.20(SAT) ~ 21(MON)<br><span>
+            @Grand Park Concert Hall, downtown Los Angeles
+        </span></p>
+    </div>
+    </section>
```

Comment:
- [ ] Good job working on the hero section, but there are still a few discrepancies that should be fixed. They are listed below
- [ ] Please kindly ensure that there is appropriate padding across the section so that it matches the original design.
- [ ] Please kindly include a background image just like that of the original design
- [ ] The font type used does not match that (Cocogoose) specified in the original design guidelines. Please kindly help fix this.
- [ ] Also, there should be background image clip on the text so that it matches the design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/219870467-2f62ec30-f17e-4741-8e3b-51f136283b91.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/219870486-dab86546-eadd-4fa7-a716-462e939df4a3.png)